src/classes/ALLO_PaymentSync_SVC.cls

```diff
@@ -409,10 +409,14 @@ public class ALLO_PaymentSync_SVC {
                 Allocation__c newOppAllocation = new Allocation__c(
                     General_Accounting_Unit__c = gau,
                     Opportunity__c = context.opportunity.id,
-                    Amount__c = newGAUAmtForOpp,
-                    Percent__c = (context.Opportunity.Amount != 0) ? newGAUAmtFOrOpp / context.Opportunity.Amount * 100 : 0
+                    Amount__c = newGAUAmtForOpp     
                 );
 
+                //if the GAU is default do not enter a percent
+                if (gau != UTIL_CustomSettingsFacade.getAllocationsSettings().Default__c) {
```

Comment:
Please also confirm that Default Allocations are not Enabled (getAllocationSettings().Default_Allocations_Enabled__c)In other words.

Meaning, it can set a percent if either a) It is not mapped to the Default Allocation or B) Default Allocations are not enabled.  

Basically, the value in the Default__c only matters if they have enabled Default Allocations.